ui/src/components/common/Header.js

```diff
@@ -18,8 +19,17 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { toggleHeaderDropdown } from '../../actions/actions-ui';
 
-class Header extends React.Component {
-  onTitleClick = () => {};
+export class Header extends React.Component {
+  constructor (props) {
+    super(props);
+    this.onTitleClick = this.onTitleClick.bind(this);
+    this.onUserDropdownToggle = this.onUserDropdownToggle.bind(this);
+    this.onUserDropdownSelect = this.onUserDropdownSelect.bind(this);
+    this.onLogoutUser = this.onLogoutUser.bind(this);
```

Comment:
The `this` binding being done here is not necessary as `this` is being bound below by using arrow functions to define the handlers. 

If all `bind` calls are removed, this constructor can be eliminated too